virttest/qemu_storage.py

```diff
@@ -127,13 +128,24 @@ def create(self, params, ignore_errors=False):
             qemu_img_cmd = qemu_img_cmd.rstrip(",")
 
             if self.base_tag:
+                if params.get("image_json_file.driver"):
+                    jdict = {}
+                    keys = [key for key in params.keys()
+                            if key.startswith('image_json')]
+                    for key in keys:
+                        jdict[key.split("image_json_")[1]] = params[key]
+                    jstring = json.dumps(jdict)
+                    self.base_image_filename = "\'json: {}\'".format(jstring)
                 qemu_img_cmd += " -b %s" % self.base_image_filename
                 if self.base_format:
                     qemu_img_cmd += " -F %s" % self.base_format
 
             qemu_img_cmd += " %s" % self.image_filename
 
             qemu_img_cmd += " %s" % self.size
+        command_prefix = params.get("qemu_img_command_prefix")
```

Comment:
prefix  should be applied in __init__ function, so that all sub command can used it.